### ISSUE TITLE
#lc and #lichsettings not working

### ISSUE THREAD

**Issue by Kiriawen** [GenieClient/Genie4#52]
when issuing these commands an error is generated.  It does not crash the client but the error is generated each time

after updating the settings I can now view the settings via #lc but they are not updated with what was input
```
Date and Time:         1/29/2022 10:56:31
Machine Name:          redacted
Operating System:      Microsoft Windows 10.0.22000 x64
Environment Version:   6.0.1
Genie Version:         4.0.1.2
Genie Build Date:      1/1/0001 00:00:00

SendText
Object reference not set to an instance of an object.

System.NullReferenceException: Object reference not set to an instance of an object.
   at GenieClient.Genie.Command.ParseCommand(String sText, Boolean bSendToGame, Boolean bUserInput, String sOrigin, Boolean bParseQuickSend)
   at GenieClient.FormMain.TextBoxInput_SendText(String sText)
```

**Comment by Kiriawen** [comment]
because the path cannot be updated anyone who does not have the lich or ruby folders in the default locations cannot sign in and you cannot sign into any other instance besides DR Prime

**Comment by jingounchained** [comment]
Newest version should resolve this, goes back to the intended mode of handling those settings.
https://github.com/GenieClient/Genie4/releases/tag/4.0.1.3

**Comment by Kiriawen** [comment]
Issue was resolved with update to 4.0.1.3
